frontend/src/app/components/Restart.js

```diff
@@ -13,30 +13,21 @@ export default class Restart extends React.Component {
     });
   }
 
-  renderSubtitle() {
-    const { experimentTitle } = this.props;
-    if (experimentTitle) {
-      return (
-        <h2 data-l10n-id="restartRequiredFromExperiment">
-           data-l10n-args={JSON.stringify({ experimentTitle })}
-          Please restart Firefox <br/> to enable { experimentTitle }.
-        </h2>
-      );
-    }
-    return (
-      <h2 data-l10n-id="restartRequiredFromLanding">
-        Please restart Firefox <br/> to choose your features.
-      </h2>
-    );
-  }
-
   render() {
     return (
       <div className="full-page-wrapper space-between">
-        <Header {...this.props} forceHideSettings={ false } />
-        <div className="centered-banner restart-message">
-          <p data-l10n-id="restartRequiredSubHeader">Almost done . . .</p>
-          { this.renderSubtitle() }
+        <Header {...this.props} forceHideSettings={ true } />
+        <div className="split-banner restart-message responsive-content-wrapper">
+          <div className="restart-image">
+          </div>
```

Comment:
Use an `<img>` or `<picture>` tag here? This is a content image, we shouldn't be hiding it as a background image.